documentation/cgroups_in_storm.md

```diff
@@ -0,0 +1,65 @@
+# CGroups in Storm
+
+CGroups are used by Storm to limit the resource usage of workers to guarantee fairness and QOS.  
+
+**Please note: CGroups is currently supported only on Linux platforms (kernel version 2.6.24 and above)** 
+
+## Setup
+
+To use CGroups make sure to install cgroups and configure cgroups correctly.  For more information about setting up and configuring, please visit:
+
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Resource_Management_Guide/ch-Using_Control_Groups.html
+
+A sample/default cgconfig.conf file is supplied in the <stormroot>/conf directory.  The contents are as follows:
+
+```
+mount {
+	cpuset	= /cgroup/cpuset;
+	cpu	= /cgroup/storm_resources;
+	cpuacct	= /cgroup/cpuacct;
+	memory	= /cgroup/storm_resources;
+	devices	= /cgroup/devices;
+	freezer	= /cgroup/freezer;
+	net_cls	= /cgroup/net_cls;
+	blkio	= /cgroup/blkio;
+}
+
+group storm {
+       perm {
+               task {
+                      uid = 500;
+                      gid = 500;
+               }
```

Comment:
Can we tell users what dose `uid` and `gid` mean? and how to set them?   Because it  depends on the user who runs storm.  

We can check `/etc/passwd` to find user's uid and gid. Such as in my `/etc/passwd`  the `admin`'s uid and gid both are `505`